docs/api_v2/api_v2_reference.apib

```diff
@@ -1995,7 +1995,70 @@ parameter. You can also filter runs by status ([available statuses](https://docs
             - count: 2 (number, required)
             - items (array[RunShort1,RunShort2], required)
 
-## Run object [/v2/actor-runs/{runId}{?token,waitForFinish}]
+## Run object [/v2/actor-runs/{runId}{?token,waitForFinish,status}]
+
+### Run storage endpoints
+
+This is not a single endpoint, but an entire group of endpoints that lets you to
+retrieve the last run or any of its default storages.
```

Comment:
not the "last run" but "run object"

Reply:
also 'that lets you to retrieve' => 'lets you retrieve'

Reply:
`This is not a single endpoint, but an entire group of endpoints lets you retrieve the last run or any of its default storages.` Does not make sense or does it? It probably should be something like `
This is not a single endpoint, but an entire group of endpoints letting you retrive the last run or any of its default storages.`